pkg/tsdb/cloudwatch/request_parser_test.go

```diff
@@ -322,3 +323,223 @@ func getBaseJsonQuery() *simplejson.Json {
 		"period":     "900",
 	})
 }
+
+func Test_migrateAliasToDynamicLabel_single_query_preserves_old_alias_and_creates_new_label(t *testing.T) {
```

Comment:
Can this and the following tests be inside `t.Run("Query migration ", func(t *testing.T) { ...`? That way we could get human readable test descriptions.

Reply:
Done